system/jaeger-operator/templates/operator.yaml

```diff
@@ -16,11 +17,11 @@ spec:
       serviceAccountName: jaeger-operator
       containers:
       - name: jaeger-operator
-        image: {{ .Values.global.dockerHubMirrorAlternateRegion }}/jaegertracing/jaeger-operator:{{ $options.jaeger.version }}
+        image: {{ .Values.global.dockerHubMirrorAlternateRegion }}/jaegertracing/jaeger-operator:1.30.0
         ports:
         - containerPort: 8383
           name: metrics
-        args: ["start"]
+        args: ["start", "--jaeger-all-in-one-image=keppel.eu-de-1.cloud.sap/ccloud-dockerhub-mirror/jaegertracing/all-in-one"]
```

Comment:
please use the common reference to keppel mirror, similar to https://github.com/sapcc/helm-charts/blob/master/openstack/designate/templates/api-deployment.yaml#L116